project/profiles/capacity4more/modules/c4m/content/c4m_content_project/c4m_content_project.module

```diff
@@ -392,26 +392,6 @@ function c4m_content_project_c4m_offcanvas_trigger_label_left_alter(&$trigger_la
 }
 
 /**
- * Implements hook_c4m_cta_block().
```

Comment:
@DavidHernandez 
revert this one since it doesn't display the button on the homepage ....